File: atividade1.c
//Importando bibliotecas
#include <stdio.h>

//Definindo função principal
int main()
{
    //Definindo variáveis
    int a, b, c;
    
    //Entrada de dados
    printf("Digite o valor 1 ");
    scanf("%d",&a);
    printf("Digite o valor 2 ");
    scanf("%d",&b);
    
    //Condicional para definir maior número e fazer cálculo
    if(a>b){
        c=a/b;
    }else{
        c=b/a;
    }
    //Mostrando Resultado
    printf("%d",c);

    return 0;
}

File: atividade1.py
#Entrada de dados
num1 = int(input("Digite o primeiro número: "))
num2 = int(input("Digite o segundo número: "))

#Condicional para verificar qual número é maior
if(num1 > num2):
    resultado = num1 / num2
else:    
    resultado = num2 / num1

#Saída de dados
print("O resultado da divisão é: ", resultado)